cfgov/unprocessed/apps/rural-or-underserved-tool/js/common.js

```diff
@@ -400,7 +400,7 @@ function generateCSV() {
   }
 
   // loop through each row
-  [].slice.call( DT.getEls( '.table tbody tr td' ) )
```

Comment:
This was meant to be `.o-table`. It was migrated with the original [markup](https://github.com/cfpb/rural-or-underserved-test/blob/master/src/index.html#L746), but then the table got migrated to `o-table` during the [course of the PR](https://github.com/cfpb/cfgov-refresh/pull/5092/commits/e9a816701e19fc68c9251fbb01ee121cd436bc8b) and this was missed in the update. Do we want to target `table` or `.o-table` (which is assigned to the parent div in scrollable tables). 

OR do we want to give the address tables a custom class so that if this page has other tables in the future they won't get pulled in?

Reply:
I'd think a custom class or id would be best

Reply:
Probably a custom class, because there are multiple tables.